inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/util/FileUtils.java

```diff
@@ -56,7 +56,7 @@
             }
             br.close();
         } catch (IOException e) {
-            log.error("Could not read file " + r.getName(), e);
+            log.error("Resource file [{}] not present.", r.getName());
```

Comment:
Since the code works even without the resources, I would make it a warning. Please also mention which functionality is not available when the resources is missing. E.g.

```
Resource file [{}] is missing - {}, r.getName(), reason
```

to print something lile

```
WARN - Resource file [/some/path/stopwords.txt] is missing - using entity linking support without stopwords may have a negative impact on the suggestion ranking.
```

Reply:
Addressed in new commit.